.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Summary
+
+Provide a summary of your blog post here.
+
+# Checklist
+
+[ ] I can successfully knit the Rmd file.
+
+[ ] Figures, graphs, tables render properly in the Rmd file.
+
+[ ] Abbreviations are expanded/explained.
+
+[ ] The Rmd file header contains the post title, post description, author information, and categories.
```

Comment:
Wondering if we should add a checklist item for making sure there is no PHI included in the blog post?

Reply:
Yes, good call! I'll update to include that.